apps/ledger-live-mobile/src/reducers/accounts.ts

```diff
@@ -262,7 +263,7 @@ export const parentAccountSelector = createSelector(
 export const accountScreenSelector =
   (route?: {
     params?: {
-      account?: AccountLike;
+      account?: AccountLike | CosmosAccount;
```

Comment:
[Question]: Why the need to add `CosmosAccount` here?

Reply:
I think `AccountLike` it is the common type provided by any coin, it is failing because we need to use specific CosmosAccount  properties.

Reply:
Perhaps it might be better to leave this as is since it seems quite generalised.

One idea is to use `getAccountCurrency` helper along with the `accountScreenSelector`?